### PR TITLE
Use bignumber for number comparisons in useGasFeeInput

### PR DESCRIPTION
**Pull request by danjm**
This PR ensures that the number comparisons we do to set errors in useGasFeeInputs will be valid for all inputs. It can prevent errors when comparing stringified numbers, or highly precise numbers.

### CODE REVIEW

**Review by dan437 — DISMISSED** [review]
Looks good. If we can add UTs in some future PR for these util functions, that would be great.

File: ui/helpers/utils/util.js [hunk 1]
```diff
@@ -345,3 +345,31 @@ export function constructTxParams({
...
+    return null;
+  }
+  return new BigNumber(a, 10).gt(b, 10);
+}
+
+export function bnLessThan(a, b) {
+  if (a === null || a === undefined || b === null || b === undefined) {
+    return null;
+  }
+  return new BigNumber(a, 10).lt(b);
```
Comment (darkwing) [inline ui/helpers/utils/util.js:360]: Does `b` need `,10`?

File: ui/helpers/utils/util.js [hunk 2]
```diff
@@ -345,3 +345,31 @@ export function constructTxParams({
...
+}
+
+export function bnGreaterThanEqualTo(a, b) {
+  if (a === null || a === undefined || b === null || b === undefined) {
+    return null;
+  }
+  return new BigNumber(a, 10).gte(b);
```
Comment (darkwing) [inline ui/helpers/utils/util.js:367]: Does `b` need `,10`?

File: ui/helpers/utils/util.js [hunk 3]
```diff
@@ -345,3 +345,31 @@ export function constructTxParams({
...
+}
+
+export function bnLessThanEqualTo(a, b) {
+  if (a === null || a === undefined || b === null || b === undefined) {
+    return null;
+  }
+  return new BigNumber(a, 10).lte(b);
```
Comment (darkwing) [inline ui/helpers/utils/util.js:374]: Does `b` need `,10`?

**Comment by darkwing** [comment]
Do we want to return `false` instead of null?  I guess I expect a boolean value from these functions.  We could also trigger an error to sentry or something?

**Comment by darkwing** [comment]
Do we want to return `false` instead of null?  I guess I expect a boolean value from these functions.  We could also trigger an error to sentry or something?

QA'd and it works but love to know if we should be adding the second param for `b`.

**Review by darkwing — DISMISSED** [review]

**Review by dan437 — DISMISSED** [review]

**Review by darkwing — APPROVED** [review]

**Review by dan437 — APPROVED** [review]

**Pull request merged**